apps/explorer/src/core/components/ui/AppLiveUpdate.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-btn v-if="update" block flat class="update-btn" @click="refresh">
+    <v-layout align-center justify-center row wrap fill-height>
+      <p class="text-lowercase font-weight-regular pr-2 mb-0">{{ formatStr(newTxs.toString()) }} {{ $t('message.updateTxs') }}</p>
+      <v-icon class="secondary--text" small> fas fa-sync-alt</v-icon>
+    </v-layout>
+  </v-btn>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Mixins } from 'vue-property-decorator'
+import { Events, BlockMetrics } from 'ethvm-common'
+import { StringConcatMixin } from '@app/core/components/mixins'
+
+@Component
+export default class AppLiveUpdate extends Mixins(StringConcatMixin) {
+  lastBlock: string = null
+  update = false
+  newTxs = 1000000
+  /*
```

Comment:
Add space between variable newTxs and comment

Reply:
Done